crates/project_panel/src/project_panel.rs

```diff
@@ -7280,6 +7280,83 @@ mod tests {
         );
     }
 
+    #[gpui::test]
```

Comment:
```suggestion
    #[gpui::test]
    #[cfg_attr(target_os = "windows", ignore)]
```
We could add this to skip the test on Windows